doc/nrf/gs_assistant.rst

```diff
@@ -1,19 +1,19 @@
 .. _gs_assistant:
 
-Installing the |NCS| through nRF Connect for Desktop
-####################################################
+Installing automatically
+########################
 
 .. contents::
    :local:
    :depth: 2
 
-`nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS, provides different apps that simplify installing the |NCS|.
+`nRF Connect for Desktop`_, a cross-platform tool available for Windows and macOS, provides different apps that simplify installing the |NCS|.
```

Comment:
I don't think removing Linux here is correct since nRF Connect for Desktop is available on Linux. It just doesn't simplify the installation as much on Linux.

Reply:
Ah yes, good point. I will add that back. I think the next paragraph addresses the Linux vs other OS updates I was thinking of.